Backend/lean_fridge_flask.py

```diff
@@ -30,12 +30,17 @@ def itemDisplay(item_id):
 @app.route('/stock/new/', methods = ['GET', 'POST'])
 def newItem():
     if request.method == 'POST':
+        stock = session.query(Stock).all()
```

Comment:
instead of reading all the stock, a more efficient way would be to query the DB for this one item and see if it exists?